### PR TITLE
Issue 7 : Added UT wrapper for CU_ASSERT_EQUAL

### PR DESCRIPTION
**Pull request by AppleNinja910**

### CODE REVIEW

File: include/ut.h [hunk 1]
```diff
@@ -152,6 +152,7 @@ UT_test_t *UT_add_test( UT_test_suite_t *pSuite, const char *pTitle, UT_TestFunc
 #define UT_ASSERT_TRUE(value) CU_ASSERT_TRUE_FATAL(value)
 #define UT_ASSERT_FALSE(value) CU_ASSERT_FALSE_FATAL(value)
 #define UT_ASSERT_EQUAL(actual,expected) CU_ASSERT_EQUAL_FATAL(actual,expected)
+#define UT_ASSERT_EQUAL_NOT_FATAL(actual,expected) CU_ASSERT_EQUAL(actual,expected)
```
Comment (Ulrond) [inline include/ut.h:155]: Do you think we should add the full list?

e.g.
`UT_ASSERT_TRUE_NOT_FATAL`
`UT_ASSERT_FALSE_NOT_FATAL`

**Review by Ulrond — CHANGES_REQUESTED** [review]
minor changes

File: include/ut.h (hunk 1 above)
Comment (AppleNinja910) [reply include/ut.h:155]: Agreed. Should we also add UT_ASSERT_NOT_FATAL?

File: include/ut.h [hunk 2]
```diff
@@ -138,7 +138,7 @@ UT_test_t *UT_add_test( UT_test_suite_t *pSuite, const char *pTitle, UT_TestFunc
 #include <TestRun.h>
 #include <CUnit.h>
 
-#define UT_VERSION "1.0.0"
```
[the remark this answers is not in the document]
Comment (AppleNinja910) [reply include/ut.h:141]: Done. Removed UT_CORE_VERSION as well as discussed

File: include/ut.h (hunk 1 above)
Comment (AppleNinja910) [reply include/ut.h:167]: Add NOT-FATAL equivalents to existing UT Test Macros as discussed

**Review by Ulrond — APPROVED** [review]
looks good

**Review by Ulrond — CHANGES_REQUESTED** [review]
UT_ASSERT(value) -> Not right on the CU_ASSERT(msg) side.

File: include/ut.h [hunk 3]
```diff
@@ -138,23 +135,44 @@ UT_test_t *UT_add_test( UT_test_suite_t *pSuite, const char *pTitle, UT_TestFunc
...
 
-/* Fatal test macros */
-#define UT_FAIL_FATAL(msg) CU_FAIL_FATAL(msg)
+#define UT_FAIL(msg) CU_FAIL_FATAL(msg)
+/* NON-FATAL equivalent of UT_FAIL test macro */
+#define UT_FAIL_NOT_FATAL(msg) CU_FAIL(msg)
+
 #define UT_ASSERT(value) CU_ASSERT_FATAL(msg)
+/* NON-FATAL equivalent of UT_ASSERT test macro */
+#define UT_ASSERT_NOT_FATAL(value) CU_ASSERT(msg)
```
Comment (Ulrond) [inline include/ut.h:147]: Duplicated the bug in UT_ASSERT_NOT_FATAL(value) -> CU_ASSERT(msg)

Where `msg` should be `value`

Comment (AppleNinja910) [reply include/ut.h]: Done

**Review by Ulrond — APPROVED** [review]
ship it

**Pull request merged**